### PR TITLE
[zh] Sync /setup/upgrade/canary/

### PR DESCRIPTION
**Pull request by windsonsea**
Updated a SVG and remove two redundant PNGs.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/upgrade/canary/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ $ istioctl x precheck
 
 {{< idea >}}
 当使用基于版本的升级时，支持跨越两个小版本（例如，直接从版本 `1.15` 到 `1.17`）。
-这与原地升级相反，原地升级需要升级到每个中间次要版本释放。
+这与原地升级相反，原地升级需要升级到每个中间的次要版本。
```
Comment (hanxiaop) [inline content/zh/docs/setup/upgrade/canary/index.md:29]:
```suggestion
这与金丝雀升级相反，原地升级需要升级到每个中间的次要版本。
```

**Review by hanxiaop — COMMENTED** [review]
in-place upgrade is compared to canary upgrade

File: content/zh/docs/setup/upgrade/canary/index.md (hunk 1 above)
Comment (hanxiaop) [reply content/zh/docs/setup/upgrade/canary/index.md:29]: How about
```suggestion
这与原地升级不同，原地升级要求必须升级到每一个中间的小版本。
```
?

Comment (windsonsea) [reply content/zh/docs/setup/upgrade/canary/index.md]: ok, the latter is better

**Review by hanxiaop — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**